paket.env

```diff
@@ -34,6 +34,12 @@ PAKET_ROUTER_PORT=5000
 PAKET_BRIDGE_PORT=5001
 PAKET_FUNDER_PORT=5002
 PAKET_DB_NAME=test
+PAKET_TEST_LAUNCHER_SEED=SADO46IHYJM7LAGEZ3QPCM4OMD4EA4S23TYELW4VI4ATHFM2NP3ZQMXP
+PAKET_TEST_COURIER_SEED=SC6X3R4GVX3S3A6MKF4L2KWVO5AO2DYJCEFQY57A7Y27BAKCGCMOR3YJ
+PAKET_TEST_RECIPIENT_SEED=SDCZH3DU3RYZZHCHHYC2DXSDCH2ZVK7ROR6AXLLHH6GSYUEV2SFCGHM7
+PAKET_SIMULATION_XLM_START_BALANCE=5000000000
+PAKET_SIMULATION_BUL_START_BALANCE=5000000000
+PAKET_ROUTER_URL=http://127.0.0.1
```

Comment:
@yaroslavr-itd , why do we need the router's URL? It's the manager that's running the router, it knows it is on `//localhost:<PAKET_ROUTER_PORT>`, no?

Reply:
Full URL to any endpoint in API contains:
- hostname (or IP address)
- port
- version
- endpoint name

Yes, we can use `localhost` as hostname. Yes, manager already know about port. And we know all endpoint names. But how to know version? I've resolve this problem in simple manner: just added debug-only env variable PAKET_ROUTER_URL. It can contains hostname, port and version. Yes, I've not included it to commit, there is only address, because anyway it will be modified before running.

If you have better solution, I will happy to see it.

Reply:
I will think it over.

Reply:
I always hated like this versioning thing in the URL... I guess we need to add a version number for our servers in `paket.env` at some stage. For now, leave it as it is, but please add a nice remark over the bit that uses it in the code with a short explanation and a link to this thread: https://github.com/paket-core/manager/pull/11#discussion_r227434755